src/database.js

```diff
@@ -121,13 +92,12 @@ const Database = async ({ ipfs, identity, address, name, access, directory, meta
   const log = await Log(identity, { logId: address, access, entryStorage, headsStorage, indexStorage })
 
   /**
-   * Event emitter that emits updates.
-   * @name events
+   * Event emitter that emits Database changes.
    * @†ype EventEmitter
-   * @fires update when an entry is added to the database.
-   * @fires close When the database is closed.
-   * @fires drop When the database is dropped.
-   * @memberof module:Database~Database
+   * @fires update when addOperation or applyOperation completes.
```

Comment:
I would change this back to "...when an entry is added to the database" because "addOperation" and "applyOperation" are internals of Database whereas the user would expect that "an entry was added" without having to look into the internals.